src/octotree.js

```diff
@@ -190,4 +190,10 @@ $(document).ready(() => {
       return $toggler.is(':visible')
     }
   }
-})
+}
+
+if(document.readyState === 'complete') {
```

Comment:
`$(document).ready(...)` not work? I'm quite surprised if that's the case.

nitpick: need a space after `if`.

Reply:
I don't know the reason why `$(document).ready(...)` fails either.